bin/aws-iam-login.inc.sh

```diff
@@ -61,4 +67,10 @@ function _aws_profile_completer() {
     COMPREPLY=($(compgen -W "${AWS_PROFILES}" -- "${WORD}"))
 }
 
+if [ -n "$ZSH_VERSION" ]; then
+    autoload -U compaudit compinit bashcompinit && bashcompinit || {
+        echo >&2 "Initialization of Zsh completion features has failed in aws-iam-login.inc.sh."
+        return 1
+    }
+fi
 complete -F _aws_profile_completer aws-iam-login
```

Comment:
Won't this line fail on zsh? Wasn't this a case?

Reply:
`bashcompinit` enables support of Bash autocompletion builtins within Zsh so that we should be able to call `complete` after having called `bashcompinit`.

No problem on my terminal so far.

Reply:
Awesome! Thanks!

Reply:
I'm happy I was able to make this small contribution :-)